apps/certificate/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CertificateConfig(AppConfig):
```

Comment:
I also do not see a reason for this to become a separate app - the only file used is `certificate.py`, and this app doesn't really offer any standalone features that can be plugged into different places like a Django app should.

`certificate.py` could've easily been in `client` (the `client` and `server` app are in need of name-changing and reorganizing as they will become deprecated with the introduction of the new frontends but that's a separate issue).